README.md

```diff
@@ -28,6 +27,10 @@ More about how to start WebDriverAgent [here](https://github.com/facebook/WebDri
 ## Known Issues
 If you are having some issues please checkout [wiki](https://github.com/facebook/WebDriverAgent/wiki/Common-Issues) first.
 
+## Note
+
+[appium-webdriveragent](https://www.npmjs.com/package/appium-webdriveragent) on npm registry expects build via [XCUITest driver](https://github.com/appium/appium-xcuitest-driver).
```

Comment:
hm, I don't quite get the meaning of this sentence

Reply:
I would have like to address a case a user tries to run the getting start on appium-webdriveragent which is available via npm. It has no `Script/` since xcuitest-driver runs carthage command etc. => https://github.com/appium/appium/issues/12400#issuecomment-516270353